src/analysis/retail/monk/mistweaver/modules/spells/Unison.tsx

```diff
@@ -0,0 +1,74 @@
+import { TALENTS_MONK } from 'common/TALENTS';
+import SpellLink from 'interface/SpellLink';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options, SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import Events, { HealEvent} from 'parser/core/Events';
+import { formatNumber } from 'common/format';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+
+class Unison extends Analyzer {
+healing: number = 0;
```

Comment:
This formatting seems off?

Reply:
Oh yeah it looks here like these fields didn't get indented - but it's weird because I thought the linter took care of that

Reply:
Formatting done